app/Http/Controllers/PaymentRequestController.php

```
@@ -136,32 +137,19 @@ public function store(PaymentRequestStore $request)
-                    'description' => $template->description,
-                    'status' => $status,
-                    'priority' => $template->priority,
-                ];
-            })->toArray();
-            TransactionFlow::insert($transactionFlowData);
-            TransactionLog::create([
-                'type' => TransactionLogStatus::REQUEST->value,
+            $transactionFlowData = TransactionFlowService::getTransactionFlow(
+                PaymentRequestType::PRF->value,
+                $paymentRequest->id
+            );
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Services\TransactionFlowService' in method 'store'. (Clean Code Rules)

(StaticAccess)